packages/ui/src/List/components/ListItem/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React, {createContext, FC, ReactNode, useContext} from 'react';
+import styles from './ListItem.module.scss';
+import cx from 'classnames';
+import {useListContext} from '../../index';
+import {Caption} from '../Content/Caption';
+import {MainText} from '../Content/MainText';
+import {findChildByName} from './utils';
+
+interface ListItemContextType {
+  disabled: boolean;
+}
+
+const ListItemContext = createContext<ListItemContextType | null>(null);
```

Comment:
если тут только бул лежит, может его попроще сделать?
```suggestion
const ListItemDisabledContext = createContext<boolean>(false);
```